src/ucp/rndv/rndv.c

```diff
@@ -433,8 +433,6 @@ ucp_rndv_progress_rma_zcopy_common(ucp_request_t *req, ucp_lane_index_t lane,
     ucs_assert_always(req->send.lane != UCP_NULL_LANE);
     ucs_assert_always(req->send.rndv.lanes_count > 0);
 
-    req->send.pending_lane = UCP_NULL_LANE;
```

Comment:
pls, add initialization in `ucp_rndv_receive()` for GET and in `ucp_rndv_rtr_handler()` for PUT